[2024-05-19T20:27:13.932+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-19T20:27:13.963+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-06-20T00:00:00+00:00 [queued]>
[2024-05-19T20:27:13.985+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-06-20T00:00:00+00:00 [queued]>
[2024-05-19T20:27:13.987+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-19T20:27:14.006+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): process_parquet_files> on 2023-06-20 00:00:00+00:00
[2024-05-19T20:27:14.030+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=863) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-19T20:27:14.035+0000] {standard_task_runner.py:63} INFO - Started process 884 to run task
[2024-05-19T20:27:14.033+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'silver_s3', 'process_parquet_files', 'scheduled__2023-06-20T00:00:00+00:00', '--job-id', '300', '--raw', '--subdir', 'DAGS_FOLDER/silver_s3.py', '--cfg-path', '/tmp/tmpg0869orj']
[2024-05-19T20:27:14.039+0000] {standard_task_runner.py:91} INFO - Job 300: Subtask process_parquet_files
[2024-05-19T20:27:14.128+0000] {task_command.py:426} INFO - Running <TaskInstance: silver_s3.process_parquet_files scheduled__2023-06-20T00:00:00+00:00 [running]> on host 620090466bc8
[2024-05-19T20:27:14.337+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='silver_s3' AIRFLOW_CTX_TASK_ID='process_parquet_files' AIRFLOW_CTX_EXECUTION_DATE='2023-06-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-20T00:00:00+00:00'
[2024-05-19T20:27:14.343+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-19T20:27:14.387+0000] {base.py:84} INFO - Using connection ID 'aws_s3' for task execution.
[2024-05-19T20:27:14.389+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_s3', conn_type='aws') credentials retrieved from login and password.
[2024-05-19T20:27:19.467+0000] {logging_mixin.py:188} INFO - No .parquet files found.
[2024-05-19T20:27:19.478+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-19T20:27:19.492+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-19T20:27:19.506+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=silver_s3, task_id=process_parquet_files, run_id=scheduled__2023-06-20T00:00:00+00:00, execution_date=20230620T000000, start_date=20240519T202713, end_date=20240519T202719
[2024-05-19T20:27:19.559+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-19T20:27:19.569+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-19T20:27:19.569+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-05-19T20:40:24.987+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-19T20:40:25.005+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-06-20T00:00:00+00:00 [queued]>
[2024-05-19T20:40:25.009+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: silver_s3.process_parquet_files scheduled__2023-06-20T00:00:00+00:00 [queued]>
[2024-05-19T20:40:25.009+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-05-19T20:40:25.018+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): process_parquet_files> on 2023-06-20 00:00:00+00:00
[2024-05-19T20:40:25.023+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2612) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-19T20:40:25.026+0000] {standard_task_runner.py:63} INFO - Started process 2635 to run task
[2024-05-19T20:40:25.024+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'silver_s3', 'process_parquet_files', 'scheduled__2023-06-20T00:00:00+00:00', '--job-id', '417', '--raw', '--subdir', 'DAGS_FOLDER/silver_s3.py', '--cfg-path', '/tmp/tmpkuvlikbp']
[2024-05-19T20:40:25.027+0000] {standard_task_runner.py:91} INFO - Job 417: Subtask process_parquet_files
[2024-05-19T20:40:25.065+0000] {task_command.py:426} INFO - Running <TaskInstance: silver_s3.process_parquet_files scheduled__2023-06-20T00:00:00+00:00 [running]> on host 620090466bc8
[2024-05-19T20:40:25.124+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='silver_s3' AIRFLOW_CTX_TASK_ID='process_parquet_files' AIRFLOW_CTX_EXECUTION_DATE='2023-06-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-20T00:00:00+00:00'
[2024-05-19T20:40:25.125+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-19T20:40:25.140+0000] {base.py:84} INFO - Using connection ID 'aws_s3' for task execution.
[2024-05-19T20:40:25.140+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_s3', conn_type='aws') credentials retrieved from login and password.
[2024-05-19T20:40:32.908+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-19T20:40:32.912+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/silver_s3.py", line 67, in process_parquet_files
    os.remove(file_path)
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/disney_data_20240518T000000.parquet'
[2024-05-19T20:40:32.925+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=silver_s3, task_id=process_parquet_files, run_id=scheduled__2023-06-20T00:00:00+00:00, execution_date=20230620T000000, start_date=20240519T204025, end_date=20240519T204032
[2024-05-19T20:40:32.936+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 417 for task process_parquet_files ([Errno 2] No such file or directory: '/tmp/disney_data_20240518T000000.parquet'; 2635)
[2024-05-19T20:40:32.980+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-05-19T20:40:32.992+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-19T20:40:32.993+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
